plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -295,3 +298,7 @@ export async function createRouter(
 
   return app;
 }
+
+function getBearerToken(header?: string): string | undefined {
+  return header?.match(/Bearer\s+(\S+)/i)?.[1];
```

Comment:
```suggestion
  return header?.match(/^[bB]earer (\S+)$/)?.[1];
```

made me go peek at several RFCs to see if Bearer really should be case sensitive :) And it seems it actually should. But sure, "forgiving on input and strict on output"... I believe my suggestion makes things more explicit, and there should be exactly one space.

Reply:
Think the existing one's fine tbh, no need to be too strict, most important is that we send with the correct format